docker-compose.yml

```diff
@@ -8,53 +8,12 @@ services:
       - "./docs/openapi/:/usr/share/nginx/html/swagger/"
     environment:
       - SPEC_URL=swagger/index.yaml
-  backend:
-    build:
-      context: .
-    command: yarn server
-    user: ${CURRENT_USER:-root}
-    ports:
-      - "8080:8080"
-    depends_on:
-      - db
-      - redis
-    environment:
-      - POSTGRES_HOST=postgres_docker
-      - REDIS_HOST=redis
-    volumes:
-      - ".:/app:rw"
-  frontend:
-    build:
-      context: .
-    command: yarn start
-    user: ${CURRENT_USER:-root}
-    stdin_open: true
-    ports:
-      - "3000:3000"
-    volumes:
-      - "./frontend:/app:rw"
-      - "./scripts:/app/scripts"
-    environment:
-      - BACKEND_PROXY=http://backend:8080
-  worker:
-    build:
-      context: .
-    command: yarn worker
-    env_file: .env
-    depends_on:
-      - db
-      - redis
-    environment:
-      - POSTGRES_HOST=postgres_docker
-      - REDIS_HOST=redis
-    volumes:
-      - ".:/app:rw"
   db:
     image: postgres:12.4
     container_name: postgres_docker
     env_file: .env
     ports:
-      - "6543:5432"
+      - "5432:5432"
```

Comment:
Yeah, I was trying to keep the PORT configuration consistent across any possible invocation. Since postgres will be listening on 5432 no matter what we put here when going `docker <-> docker` this seemed the best way to handle that.

I'll add a README warning